src/south/SQLDbToFile/SQLDbToFile.class.js

```diff
@@ -93,6 +93,26 @@ class SQLDbToFile extends ProtocolHandler {
     }
   }
 
+  /**
+   * Function used to parse an entry and update the lastCompletedAt if needed
+   * @param {*} entry - on sql result item
+   * @return {void}
+   */
+  checkLastCompletedAt(entry) {
+    if (entry[this.timeColumn] && (entry[this.timeColumn] instanceof Date) && (entry[this.timeColumn] > new Date(this.lastCompletedAt))) {
+      this.lastCompletedAt = entry[this.timeColumn].toISOString()
+    } else if (entry[this.timeColumn]) {
+      const entryDate = new Date(entry[this.timeColumn])
+      if (entryDate.toString() !== 'Invalid Date') {
+        // We need to take back the js added timezone since it is not in the original string coming from the database
+        const entryDateWithouthTimezonOffset = new Date(entryDate.getTime() - entryDate.getTimezoneOffset() * 60000)
```

Comment:
Without not Withouth

Reply:
pour passer en UTC, il n'y a pas la fonction Date.UTC()?

Reply:
Date.UTC takes in argument several number (year, month, day, etc... : https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC)
Numbers that I don't have without parsing the string date into a date format which is what I do line 105. And the Date constructor give an offset depending of the runtime environment timezone. That is why I remove the offset in the constant.